docs/book/src/developer/tilt.md

```diff
@@ -10,7 +10,7 @@ This document describes how to use [kind](https://kind.sigs.k8s.io) and [Tilt](h
 2. [kind](https://kind.sigs.k8s.io) v0.9 or newer (other clusters can be
    used if `preload_images_for_kind` is set to false)
 3. [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
-4. [Tilt](https://docs.tilt.dev/install.html) v0.22.2 or newer
+4. [Tilt](https://docs.tilt.dev/install.html) v0.30.8 or newer
```

Comment:
any reason for bumping the minimum tilt version required?

Reply:
Its the requirement from cluster api: https://github.com/Karthik-K-N/cluster-api/blob/3aae0e86340a519feffc3e398c2fa81795fd73c9/Tiltfile#L19